swagger/v1/openapi.json

```diff
@@ -2535,6 +2535,37 @@
             "$ref": "#/components/schemas/relationship_collection"
           }
         }
+      },
```

Comment:
Regenerate this file (with automatic examples) like this:

```
podman exec -it compliance-backend_rails_1 /usr/local/bin/ruby -I/usr/local/bundle/gems/rspec-core-3.9.2/lib:/usr/local/bundle/gems/rspec-support-3.9.3/lib /usr/local/bundle/gems/rspec-core-3.9.2/exe/rspec --pattern spec/requests/\*\*/\*_spec.rb,\ spec/api/\*\*/\*_spec.rb,\ spec/integration/\*\*/\*_spec.rb --format Rswag::Specs::SwaggerFormatter --order defined
```

Sorry, it's pretty hacky; something we need to improve. Last time I looked into it, there were some issues with rswag preventing a simpler command, but I can't remember the details.

Reply:
I've added the spec and run the command.

`bundle exec rspec +args` worked for me. We might add a rake task or improve `rswag:specs:swaggerize`.